### PR TITLE
http: let configure global directories scan via THttpServer constructor

### PR DESCRIPTION
**Pull request by linev**
Simplifies usage for the use-case:

https://root-forum.cern.ch/t/thttpserver-without-items-from-groot-only-explicitly-registered/24640

### CODE REVIEW

File: documentation/HttpServer/HttpServer.md [hunk 1]
```diff
@@ -70,7 +80,7 @@ For that one should use `%arg1`, `%arg2` and so on identifiers. Like:
     serv->RegisterCommand("/DoSomething","SomeFunction(%arg1%,%arg2%)");
 
 User will be requested to enter arguments values, when command element clicked in the browser.
-Example of the command which executes arbitrary string in application via ProcessLine looks like:
+Example of the command which executes arbitrary string in appliction via ProcessLine looks like:
```
Comment (bellenot) [inline documentation/HttpServer/HttpServer.md]: There are several typo. This is the first one...

File: documentation/HttpServer/HttpServer.md [hunk 2]
```diff
@@ -142,7 +151,7 @@ One could specify a debug parameter to be able to adjust the FastCGI configurati
 
 All user access will be ruled by the main web server. Authorized account names could be used to configure access restriction in THttpServer.
 
-### Configure fastcgi with Apache2
+### Configure fastcgi with Apcahe2
```
Comment (bellenot) [inline documentation/HttpServer/HttpServer.md]: second one

File: documentation/HttpServer/HttpServer.md [hunk 3]
```diff
@@ -387,7 +396,7 @@ One could specify them in the URL string:
 
 ### Performing multiple requests at once
 
-To minimize traffic between sever and client, one could submit several requests at once. This is especially useful when big number of small objects should be requested simultaneously. For this purposes `multi.bin` or `multi.json` requests could be used.
+To minimize traffic between sever and client, one could submit several requests at once. This is especially useful when big number of small objects should be requestsed simultaneosely. For this purposes `multi.bin` or `multi.json` requests could be used.
```
Comment (bellenot) [inline documentation/HttpServer/HttpServer.md]: third

File: documentation/HttpServer/HttpServer.md [hunk 4]
```diff
@@ -405,7 +414,7 @@ For `multi.bin` any kind of requests can be used. It returns binary buffer with
     [size2 (little endian), 4 bytes] + [request2 result, size2 bytes]
     [size3 (little endian), 4 bytes] + [request3 result, size3 bytes]
 
-While POST data in request used to transfer list of multiple requests, it is not possible to submit
+While POST data in request used to transfer list of multiple reqeusts, it is not possible to submit
```
Comment (bellenot) [inline documentation/HttpServer/HttpServer.md]: another one

File: documentation/HttpServer/HttpServer.md [hunk 5]
```diff
@@ -419,5 +428,4 @@ To use `multi.json` request from the JavaScript, one should create special 'POST
      });
      xhr.send("Files/job1.root/hpx/root.json\nFiles/job1.root/hpxpy/root.json\nFiles/job1.root/hprof/root.json\n");
 
-Here argument "multi" identifies, that server response should be parsed with `JSROOT.parse_multi()` function, which correctly interprets JSON code, produced by `multi.json` request. When sending such request to the server, one should provide list of objects names and not forget "?number=N" parameter in the request URL string.
-
+Here arguemnt "multi" identifies, that server response should be parsed with `JSROOT.parse_multi()` function, which correctly interprets JSON code, produced by `multi.json` request. When sending such request to the server, one should provide list of objects names and not forget "?number=N" parameter in the request URL string.
```
Comment (bellenot) [inline documentation/HttpServer/HttpServer.md]: and yet another one... Could you review and fix them, please?

**Comment by linev** [comment]
I hope, now I fix all spelling errors

File: documentation/HttpServer/HttpServer.md [hunk 6]
```diff
@@ -14,7 +14,7 @@ To start the http server, at any time, create an instance of the [THttpServer](h
 
 This will start a [civetweb](https://github.com/civetweb/civetweb)-based http server on the port 8080. Then one should be able to open the address "http://localhost:8080" in any modern browser (Firefox, Chrome, Opera, Safari, IE11) and browse objects created in application. By default, the server can access files, canvases, and histograms via the gROOT pointer. All those objects can be displayed with JSROOT graphics.
 
-There is a [server snapshot](https://root.cern/js/latest/httpserver.C/?layout=simple&item=Canvases/c1) of running macro [tutorials/http/httpserver.C](https://github.com/root-project/root/blob/master/tutorials/http/httpserver.C) from ROOT tutorials.
+There is a [server snapshot](https://root.cern/js/latest/httpserver.C/?layout=simple&item=Canvases/c1) of running macro [tutorials/http/httpserver.C](https://github.com/root-mirror/root/blob/master/tutorials/http/httpserver.C) from ROOT tutorials.
```
Comment (bellenot) [inline documentation/HttpServer/HttpServer.md]: The right URL is https://github.com/root-project (and not https://github.com/root-mirror)

Comment (linev) [inline documentation/HttpServer/HttpServer.md]: Now also corrected.

Bertrand, you can edit HttpServer.md file directly and I will update it in JSROOT.

Comment (bellenot) [inline documentation/HttpServer/HttpServer.md]: Thanks Sergey

**Pull request merged**